doc/manual/src/release-notes/release-notes.md

```diff
@@ -1 +1,11 @@
 # Nix Release Notes
+
+Nix has a release cycle of roughly 6 weeks.
+Notable changes and additions are announced in the release notes for each version.
+
+Bugfixes can be backported on request to previous Nix releases.
+We typically backport only as far back as the Nix version used in the latest NixOS release, which is announced in the [NixOS release notes](https://nixos.org/manual/nixos/stable/release-notes.html#ch-release-notes).
+
+Backports never skip releases (if a feature is backported to `x.y`, it must also be available in `x.(y+1)`;
```

Comment:
I'm not sure this is actually a policy we're currently following. Backports are pretty much by request.

Reply:
It's not, but @Roberth would like to make it such. Declaring it here may be a good start.

Reply:
It's nice in theory, but also potentially a lot of extra work, which might make us more reluctant to backport bug fixes.

Reply:
I think it's really important. If backports skip releases, we are introducing breaking changes.